scripts/technical-debt-metrics.sh

```diff
@@ -160,12 +160,14 @@ then
   then
     printf '<details><summary>No changes to technical debt.</summary>\n'
   else
-    printf '%s\n' "${rep}" |
+    printf '%s\n' "${rep}" |  # outputs lines containing `|Current number|Change|Type|`, so
+                              # `$2` refers to `Current number` and `$3` to `Change`.
       awk -F '|' -v rep="${rep}" '
-        BEGIN{total=0; weight=0}
+        BEGIN{total=0; weight=0; absWeight=0}
+        {absWeight+=$3+0}
         (($3+0 == $3) && (!($2+0 == 0))) {total+=1 / $2; weight+=$3 / $2}
         END{
-          average=weight/total
+          if (total == "0") {average=absWeight} else {average=weight/total}
```

Comment:
I also am just looking at the awk manual while reviewing this, but is there a reason to compare with "0" rather than just 0 here?

Reply:
OK, thanks! I think since `total` should only ever be a number, the quotes here are probably introducing an unncessary coercion, which was confusing at least to me, so let's just get rid of them:
```suggestion
          if (total == 0) {average=absWeight} else {average=weight/total}
```

bors d+